frontend/src/scenes/heatmaps/HeatmapsBrowser.tsx

```diff
@@ -211,26 +234,28 @@ function EmbeddedHeatmapBrowser({
     return browserUrl ? (
         <div className="flex flex-row gap-x-2 w-full">
             <FilterPanel />
-            <iframe
-                ref={iframeRef}
-                className="flex-1"
-                src={appEditorUrl(browserUrl, {
-                    userIntent: 'heatmaps',
-                })}
-                // eslint-disable-next-line react/forbid-dom-props
-                style={{
-                    background: '#FFF',
-                }}
-                onLoad={onIframeLoad}
-                // these two sandbox values are necessary so that the site and toolbar can run
-                // this is a very loose sandbox,
-                // but we specify it so that at least other capabilities are denied
-                sandbox="allow-scripts allow-same-origin"
-                // we don't allow things such as camera access though
-                allow=""
-            />
-
-            {loading && <SpinnerOverlay />}
+            <div className="relative flex-1 w-full h-full">
+                {iframeBanner && <IframeErrorOverlay />}
```

Comment:
Feels easier to pass this value through rather than calling the logic again in the component:
```suggestion
                {iframeBanner && <IframeErrorOverlay iframeBanner={iframeBanner} />}
```